client/components/Common/Modal.tsx

```diff
@@ -9,7 +9,7 @@ export default class Modal extends React.Component<{}, State> {
     super(props)
 
     this.state = {
-      modalShown: false,
+      modalShown: true,
```

Comment:
I changed `modalShown` to preview on Storybook.
If there are some problems, please tell me!

Reply:
Uh, it's no longer used. Would you remove this component and use `UserDate` instead of it?

Reply:
Does it mean I would completely remove this component from this app or only a story file?

Reply:
@lightnet328 Should I remove `client/components/Common/Modal.tsx` ?

Reply:
@saitoeku3 It seems `Common/Modal` component added as trial of replacement from jQuery to React in past (because the value of form is fixed and it clearly not used), and actually, it is not `no longer used` but `not yet used` I think.
I'm sorry that it confuses you because of the incompleted component.

How about try `Icon` as an initial example to introduce Storybook?
`Icon` is the good simple component.

Reply:
@sotarok  I did it! Thanks for your good suggestion!

<img width="1291" alt="Screen Shot 2019-08-27 at 23 41 06" src="https://user-images.githubusercontent.com/38908416/63781277-7c202d80-c924-11e9-9728-73f7eed59764.png">

![Aug-27-2019 23-42-59](https://user-images.githubusercontent.com/38908416/63781349-94904800-c924-11e9-932f-1ed44891bb57.gif)